mkdocs.yml

```diff
@@ -0,0 +1,62 @@
+site_name: AdaptiveCpp Documentation
+
+repo_url: https://github.com/AdaptiveCpp/AdaptiveCpp
+repo_name: AdaptiveCpp
+
+copyright: Copyright(C) 2021-2023 Timothée David--Cléris
```

Comment:
Ẁhat does this copyright refer to? If it's on the published documentation, it should probably more generic (e.g. AdaptiveCpp project/contributors) since the documentation is built from preexisting text.

Reply:
I forgot to change it, it is like the other file